rest_registration/api/serializers.py

```diff
@@ -154,7 +159,7 @@ def get_fields(self):
     def validate(self, attrs):
         if self.has_password_confirm:
             if attrs['password'] != attrs['password_confirm']:
-                raise ValidationError('Passwords don\'t match')
+                raise ValidationError(_('Passwords don\'t match'))
```

Comment:
A minor thing; you could consider improving existing code by replacing all `'` to `"` for each message in places where you're changing the code so we could have a better consistency.